src/command_modules/azure-cli-storage/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.25
 ++++++
+* Adding soft-delete for blobs.
 * Minor fixes.
```

Comment:
Same here. Remove "minor fixes".  Also, recommend being more explicit on the addition:

"Added `blob service-properties delete-policy` commands to support blob soft-delete feature."